vtkext/private/module/vtkF3DImguiActor.cxx

```diff
@@ -284,6 +284,89 @@ void vtkF3DImguiActor::RenderFileName()
   ImGui::End();
 }
 
+//----------------------------------------------------------------------------
+void vtkF3DImguiActor::RenderMetaData()
+{
+  ImGuiViewport* viewport = ImGui::GetMainViewport();
+
+  constexpr float marginRight = 5.f;
+
+  ImVec2 winSize = ImGui::CalcTextSize(this->MetaData.c_str());
+  winSize.x += 2.f * ImGui::GetStyle().WindowPadding.x;
+  winSize.y += 2.f * ImGui::GetStyle().WindowPadding.y;
+
+  // it's super important to set the size of the window manually
+  // otherwise ImGui skip a frame for computing the size resulting in
+  // no UI when doing offscreen rendering
```

Comment:
not sure there is value to have this comment duplicated everywhere

Reply:
Ok, I will remove duplicates